tests_metricflow/snapshots/test_metric_filter_rendering.py/SqlQueryPlan/BigQuery/test_distinct_values_query_with_metric_filter__plan0.sql

```diff
@@ -24,7 +24,7 @@ FROM (
       FROM ***************************.dim_lux_listing_id_mapping lux_listing_mapping_src_28000
     ) subq_4
     FULL OUTER JOIN (
-      -- Pass Only Elements: ['listing', 'bookings']
+      -- Pass Only Elements: ['listing', 'listing__bookings']
```

Comment:
Like here we have the link, but on line 30 we do not. I don't think that's common with FilterElementsNode accesses - it typically matches, right?

Reply:
Yeah sorry I haven't been able to update snapshots yet! Slowly rebasing up the stack 😓

Reply:
Ok good catch on this!! I wasn't using the qualified name as the column name for group by metrics. Wasn't breaking anything, except our typical column naming convention 😉 I added a commit to fix that.